tests/CMakeLists.txt

```diff
@@ -42,9 +42,9 @@ if(QUDA_DIRAC_WILSON OR QUDA_DIRAC_CLOVER OR QUDA_DIRAC_TWISTED_MASS OR QUDA_DIR
 endif()
 
 if(QUDA_DIRAC_WILSON OR QUDA_DIRAC_CLOVER OR QUDA_DIRAC_TWISTED_MASS OR QUDA_DIRAC_TWISTED_CLOVER OR QUDA_DIRAC_DOMAIN_WALL OR QUDA_DIRAC_STAGGERED)
```

Comment:
not related to this pull specifically, but I guess this if statement should also include non-degenerate twisted mass as well.

Reply:
good catch. That has gone unnoticed for quite some time now. Fixed it now.